frontend/components/TooltipWrapper/_styles.scss

```diff
@@ -7,10 +7,7 @@
   }
 
   &__underline {
-    position: relative;
     width: fit-content;
-    // compensate for bottom border and padding to maintain centering
-    top: 2px;
     border-bottom: 1px dashed $ui-fleet-black-50;
     padding-bottom: 1px;
```

Comment:
while you're at this, can we move the `padding-bottom: 1px;` into the `@-moz-document` too? Looks slightly better on Chrome without the padding, though FF benefits from it.

Reply:
Chrome, with padding:
<img width="150" alt="image" src="https://github.com/fleetdm/fleet/assets/61553566/26ff28ed-b4a0-49c9-a969-709640239081">

Chrome, without padding:
<img width="151" alt="image" src="https://github.com/fleetdm/fleet/assets/61553566/789c47b3-8ee3-4c82-87b1-a822ae8abc32">  👍 

FF, with padding:
<img width="153" alt="image" src="https://github.com/fleetdm/fleet/assets/61553566/bb279ccc-f5f1-4255-9328-93d3d48bb610"> 👍 

FF, without padding:
<img width="149" alt="image" src="https://github.com/fleetdm/fleet/assets/61553566/a77edd5a-3b35-4dae-8a22-40cf52c1db40">

Reply:
yep will make that change